zeq/hbp/vocabulary.h

```diff
@@ -69,6 +71,17 @@ struct ImageJPEG
     const uint8_t* _data;
 };
 
+
+
+struct CellSetBinaryOp
```

Comment:
Provide some minimal doxygen documentation (e.g. the event related to this class, the purpose, ...)